src/css/components/_progress-bar.scss

```
@@ -10,3 +10,7 @@
         background: color($secondary-accent-color);
     }
 }
+
+.dark-theme .progress-bar {
+    color: inherit;
+}
```

```suggestion
}

```
No longer needed with my other suggestion